src/main/java/strategy/SupplyOperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package strategy;
```

Comment:
The `strategy` package should be under the `core.basesyntax` package to maintain the proper structure. Make sure to refactor the package declaration to reflect the project structure.